### PR TITLE
ethtool: add page

### PR DESCRIPTION
**Pull request by borland502**
Established the entrry, and added the case I needed to use the command for

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by mebeim** [comment]
Hi @borland502, I can see from [the man page](https://linux.die.net/man/8/ethtool) that this command actually has a wide variety of options and usages, while you are only suggesting one example. Are there any common usages/examples that you can think of? If so, it would be helpful to add them! Just a suggestion though, the new page is ok anyway.

File: pages/linux/ethtool.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ethtool
+
+> Displays and modifies Network Interface Controller (NIC) parameters.
```
Comment (mebeim) [inline pages/linux/ethtool.md]:
```suggestion
> Display and modify Network Interface Controller (NIC) parameters.
> More information: <http://man7.org/linux/man-pages/man8/ethtool.8.html>.
```
Let's stay impersonal, and while we're at it, also add a useful "more information" link.

File: pages/linux/ethtool.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- Set the link speed, duplex, and speed autonegotiation:
```
Comment (mebeim) [inline pages/linux/ethtool.md]:
```suggestion
- Set the link speed, duplex mode, and parameter autonegotiation for a given interface:
```
Let's clarify here. Duplex "mode" is more correct, also the autonegotiation is not just for the speed. Let's also mention that this affects the selected interface only.

**Comment by borland502** [comment]
> Hi @borland502, thank you for the contribution 🙂. I can see from [the man page](https://linux.die.net/man/8/ethtool) that this command actually has a wide variety of options and usages, while you are only suggesting one example. Are there any other common/useful examples that you know? If so, it would be helpful to add them! Just a suggestion though, the new page is ok anyway.

Thanks.  There are a ton, but I'm more comfortable adding what I've actually used -- I guess on the theory that I at least have a sample of one for a popular use case.  I'll make the changes you requested.

File: pages/linux/ethtool.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+`ethtool -s {{interface}} speed {{10/100/1000}} duplex {{half/full}} autoneg {{on/off}}`
```
[the remark this answers is not in the document]
Comment (borland502) [reply pages/linux/ethtool.md]: Curious though.  How widespread is eth0 though now?  I'm only using a few distros, but I see the new fangled enp1s0f0 name more often.

File: pages/linux/ethtool.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# ethtool
+
+> Display and modify Network Interface Controller (NIC) parameters.
+> More information: <http://man7.org/linux/man-pages/man8/ethtool.8.html>.
+
+- Set the link speed, duplex mode, and parameter autonegotiation for a given interface:
+
+`ethtool -s eth0 speed {{10|100|1000}} duplex {{half|full}} autoneg {{on|off}}`
```
Comment (mebeim) [inline pages/linux/ethtool.md]:
```suggestion
`ethtool -s {{eth0}} speed {{10|100|1000}} duplex {{half|full}} autoneg {{on|off}}`
```
Keep curly braces around editable fields.

**Pull request merged**